java/pom.xml

```diff
@@ -79,6 +80,7 @@
         <reload4j.version>1.2.20</reload4j.version>
         <java-websocket.version>1.5.2</java-websocket.version>
         <arrow.version>8.0.0</arrow.version>
+        <netty.version>4.1.86.Final</netty.version>
```

Comment:
It's not obvious why we've added a dependency on netty as this PR doesn't use add any imports of anything from petty. Presume it's needed to make the transfer manager work? Can you add a comment to explain why netty is needed?

Reply:
Will do, good point.

Reply:
Done